dashboard/src/app/billing/card-info/card-info.html

```diff
@@ -51,6 +52,7 @@
                che-label-name="State"
                che-name="state"
                che-place-holder=""
+               che-type-city
```

Comment:
@akurinnoy well this change is not urgent I approved the PR. I don't have more suitable name. Maybe che-type-city and che-type-state directives that can be based on the same class.